qemu/tests/cpu_info_check.py

```diff
@@ -0,0 +1,134 @@
+import re
+
+from avocado.utils import process
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest.utils_version import VersionInterval
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    cpuinfo query test:
+    1). run query cmd. e.g -cpu ?cpuid
+    2). check the expected info is included in the cmd output.
+    3). Boot guest and check the output of qmp command "qom-list-types"
+    4). Check the output of qmp command "query-cpu-definitions"
+    5). Check the output of qmp command "query-cpu-model-expansion"
+    """
+
+    cpu_types = ["Opteron_G5", "Opteron_G4", "Opteron_G3",
```

Comment:
Maybe we can use the property in the vt directly by call Flag.cpu_types ?

Reply:
@huangyum Do you mean the cpu.py model, I found there it does not distinguish qemu versions, do you have any suggestions to better utilize it in this case?

Reply:
Yes. How about we always keep it utmost in cpu.py and remove models in this case according to qemu version?